R/get_syllabus.R

```diff
@@ -87,7 +87,7 @@ get_timings <- function(ep) {
     }
     return(as.integer(i))
   }
-  times <- c(coerce_integer(yaml$teaching), coerce_integer(yaml$exercises))
+  times <- c(coerce_integer(yaml$teaching), coerce_integer(yaml$exercises), coerce_integer(yaml$break, 0L))
```

Comment:
> As the code highlighting suggested, `break` is a keyword in R. That means I cannot reference `my_yaml$break` in the way I do now.

That's on me, too! I should have seen that -_-. This will address that

```suggestion
  times <- c(coerce_integer(yaml$teaching), coerce_integer(yaml$exercises), coerce_integer(yaml[["break"]], 0L))
```